### ISSUE TITLE
Xunit DataAttribute extension/nuget package

### ISSUE THREAD

**Issue by jzabroski** [eoincampbell/combinatorics#16]
@eoincampbell @stephencleary I was thinking of an Xunit DataAttribute extension that calls this library to generate combinatoric input.  In the past, I've manually generated InlineData values.

Thoughts on whether this would be OK to add to this project?

Some maintainers get upset at adding more code/APIs to a project, as it means more code to support, but this project was relatively quiet until Stephen and I came along recently, so I am hopeful there is some interest in sprucing things up.

**Comment by StephenCleary** [comment]
I'd see that as a separate NuGet package (e.g., `Cominatorics.Testing`), but in the same repo is probably fine.

**Comment by jzabroski** [comment]
Yes, exactly.  I wouldn't want it side by side.  Combinatorics.Testing would have a nuget dependency id to Combinatorics, so that when you pack it, it picks it up transitively.

**Comment by eoincampbell** [comment]
@jzabroski see other comment. No issue if you want to propose some testing related extensions

**Comment by jzabroski** [comment]
Cool
